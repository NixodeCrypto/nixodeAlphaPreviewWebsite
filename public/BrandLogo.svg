<svg width="371" height="77" viewBox="0 0 371 77" fill="none" xmlns="http://www.w3.org/2000/svg">
<g clip-path="url(#clip0_301:3)">
<path d="M149.009 67H137.586L111.72 27.9157V67H100.297V9.96463H111.72L137.586 49.1306V9.96463H149.009V67ZM164.459 16.4107C162.447 16.4107 160.76 15.7851 159.401 14.534C158.095 13.2285 157.442 11.6237 157.442 9.71985C157.442 7.81595 158.095 6.23843 159.401 4.9873C160.76 3.68177 162.447 3.029 164.459 3.029C166.472 3.029 168.131 3.68177 169.437 4.9873C170.797 6.23843 171.477 7.81595 171.477 9.71985C171.477 11.6237 170.797 13.2285 169.437 14.534C168.131 15.7851 166.472 16.4107 164.459 16.4107ZM170.09 21.796V67H158.666V21.796H170.09ZM204.712 67L195.41 52.9655L187.169 67H174.93L189.699 44.3164L174.767 21.796H187.659L196.879 35.7489L205.202 21.796H217.441L202.591 44.3164L217.604 67H204.712ZM242.249 67.7344C237.898 67.7344 233.981 66.7824 230.5 64.8785C227.018 62.9202 224.271 60.1732 222.259 56.6374C220.3 53.1015 219.321 49.0218 219.321 44.398C219.321 39.7742 220.327 35.6945 222.34 32.1587C224.407 28.6228 227.209 25.903 230.744 23.9991C234.28 22.0408 238.224 21.0616 242.576 21.0616C246.928 21.0616 250.871 22.0408 254.407 23.9991C257.943 25.903 260.717 28.6228 262.73 32.1587C264.797 35.6945 265.831 39.7742 265.831 44.398C265.831 49.0218 264.77 53.1015 262.648 56.6374C260.581 60.1732 257.753 62.9202 254.162 64.8785C250.627 66.7824 246.656 67.7344 242.249 67.7344ZM242.249 57.7797C244.317 57.7797 246.248 57.2901 248.043 56.311C249.892 55.2774 251.361 53.7543 252.449 51.7416C253.537 49.7289 254.081 47.281 254.081 44.398C254.081 40.1006 252.938 36.8096 250.654 34.5249C248.424 32.1859 245.676 31.0163 242.413 31.0163C239.149 31.0163 236.402 32.1859 234.171 34.5249C231.996 36.8096 230.908 40.1006 230.908 44.398C230.908 48.6954 231.968 52.0136 234.09 54.3527C236.266 56.6374 238.986 57.7797 242.249 57.7797ZM269.641 44.2348C269.641 39.6655 270.538 35.6129 272.333 32.0771C274.183 28.5412 276.685 25.8214 279.84 23.9175C282.995 22.0136 286.504 21.0616 290.366 21.0616C293.303 21.0616 296.105 21.7144 298.77 23.0199C301.436 24.2711 303.557 25.9574 305.135 28.0789V6.61921H316.721V67H305.135V60.3092C303.721 62.5394 301.735 64.3345 299.178 65.6945C296.622 67.0544 293.657 67.7344 290.284 67.7344C286.477 67.7344 282.995 66.7552 279.84 64.7969C276.685 62.8386 274.183 60.0916 272.333 56.5558C270.538 52.9655 269.641 48.8586 269.641 44.2348ZM305.216 44.398C305.216 41.6237 304.672 39.2575 303.585 37.2992C302.497 35.2865 301.028 33.7634 299.178 32.7298C297.329 31.6419 295.343 31.0979 293.222 31.0979C291.1 31.0979 289.142 31.6147 287.347 32.6482C285.552 33.6818 284.083 35.2049 282.941 37.2176C281.853 39.1759 281.309 41.515 281.309 44.2348C281.309 46.9547 281.853 49.3481 282.941 51.4152C284.083 53.4279 285.552 54.9782 287.347 56.0662C289.197 57.1541 291.155 57.6981 293.222 57.6981C295.343 57.6981 297.329 57.1813 299.178 56.1478C301.028 55.0598 302.497 53.5367 303.585 51.5784C304.672 49.5657 305.216 47.1723 305.216 44.398ZM368.268 43.4189C368.268 45.0508 368.159 46.5195 367.942 47.825H334.896C335.168 51.0888 336.31 53.6455 338.323 55.495C340.335 57.3445 342.81 58.2693 345.748 58.2693C349.991 58.2693 353.01 56.447 354.805 52.8024H367.126C365.82 57.1541 363.318 60.7443 359.619 63.573C355.92 66.3472 351.378 67.7344 345.993 67.7344C341.641 67.7344 337.724 66.7824 334.243 64.8785C330.816 62.9202 328.123 60.1732 326.165 56.6374C324.261 53.1015 323.309 49.0218 323.309 44.398C323.309 39.7199 324.261 35.6129 326.165 32.0771C328.069 28.5412 330.734 25.8214 334.161 23.9175C337.588 22.0136 341.532 21.0616 345.993 21.0616C350.29 21.0616 354.125 21.9864 357.498 23.8359C360.925 25.6854 363.563 28.3237 365.412 31.7507C367.316 35.1233 368.268 39.0127 368.268 43.4189ZM356.437 40.155C356.382 37.2176 355.322 34.8785 353.255 33.1378C351.188 31.3427 348.658 30.4451 345.666 30.4451C342.838 30.4451 340.444 31.3155 338.486 33.0562C336.582 34.7425 335.412 37.1088 334.977 40.155H356.437Z" fill="#323232"/>
<path d="M72.5002 37.9166C72.5002 30.1978 69.9364 22.6976 65.2114 16.5939C60.4864 10.4902 53.8679 6.12882 46.3954 4.19464C38.9228 2.26045 31.0194 2.86307 23.9265 5.90783C16.8336 8.9526 10.9528 14.267 7.20788 21.0165C3.46293 27.766 2.0659 35.5683 3.23618 43.1979C4.40645 50.8275 8.07776 57.8522 13.6735 63.169C19.2693 68.4858 26.4725 71.7933 34.1519 72.5722C41.8314 73.3511 49.552 71.5572 56.1013 67.4722L49.4888 56.8706C45.2888 59.4903 40.3375 60.6407 35.4127 60.1412C30.4879 59.6417 25.8685 57.5206 22.28 54.1109C18.6914 50.7013 16.337 46.1963 15.5865 41.3035C14.836 36.4106 15.7319 31.4071 18.1336 27.0786C20.5352 22.7502 24.3065 19.342 28.8552 17.3894C33.4039 15.4368 38.4723 15.0504 43.2644 16.2908C48.0566 17.5312 52.301 20.3281 55.3311 24.2424C58.3612 28.1567 60.0054 32.9666 60.0054 37.9166H72.5002Z" fill="#82B5B2"/>
<path d="M69.3097 52.4799C72.5302 45.4826 73.3404 37.6151 71.6137 30.1083C69.887 22.6015 65.7208 15.8788 59.7668 10.9917L51.8404 20.6487C55.6589 23.7829 58.3308 28.0945 59.4382 32.9089C60.5456 37.7233 60.026 42.769 57.9606 47.2566L69.3097 52.4799Z" fill="#FD9500"/>
<path d="M69.3718 52.3443C72.1208 46.3034 73.0867 39.6028 72.1559 33.0315L59.786 34.7836C60.3829 38.9981 59.7634 43.2954 58.0004 47.1696L69.3718 52.3443Z" fill="#DC5D3C"/>
</g>
<defs>
<clipPath id="clip0_301:3">
<rect width="371" height="77" fill="white"/>
</clipPath>
</defs>
</svg>
