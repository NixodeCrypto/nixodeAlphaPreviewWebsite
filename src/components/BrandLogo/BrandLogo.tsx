import NextImg from 'next/image';

const BrandLogo = () => (
  <NextImg
    src="/BrandLogo.svg"
    alt="Nixode"
    width="150%"
    height="50%"
    layout="fixed"
  />
);

export default BrandLogo;
