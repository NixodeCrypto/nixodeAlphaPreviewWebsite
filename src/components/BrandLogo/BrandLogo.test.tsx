import { mount } from 'enzyme';
import BrandLogo from './BrandLogo';

describe('components/BrandLogo', () => {
  it('matches snapshot', () => {
    const wrapper = mount(<BrandLogo />);
    expect(wrapper).toMatchSnapshot();
  });
});
