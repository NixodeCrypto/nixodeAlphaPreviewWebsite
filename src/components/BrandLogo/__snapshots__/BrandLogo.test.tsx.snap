// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/BrandLogo matches snapshot 1`] = `ReactWrapper {}`;
